src/xia2/Modules/MultiCrystal/ScaleAndMerge.py

```diff
@@ -602,11 +593,39 @@ def run(self) -> None:
                 cluster_identifiers = [
                     self._data_manager.ids_to_identifiers_map[l] for l in s_c.labels
                 ]
-                self._scale_and_report_cluster(
-                    self._data_manager, cluster_dir, cluster_identifiers, s_c
+                subclusters.append(
+                    (cluster_dir, "coordinate", cluster_identifiers, s_c)
                 )
                 count += 1
 
+        with (
+            record_step("dials.scale(parallel)"),
+            concurrent.futures.ProcessPoolExecutor(
+                max_workers=self._params.nproc
```

Comment:
I think we want to limit max workers to be `min(self._params.nproc, len(subclusters))` to avoid spinning up more ProcessPool's than needed? Also, do we then need to restrict nproc to 1 for the programs used within `scale_and_report_cluster`, to avoid further process spawning?